src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ApartmentValueFunction.java

```diff
@@ -74,27 +44,46 @@
    * to set those two.
    */
   public ApartmentValueFunction() {
-    this.floorAreaValueFunction = new ConstantValueFunction<>(0d);
-    this.nbBedroomsValueFunction = new ConstantValueFunction<>(0d);
-    this.nbSleepingValueFunction = new ConstantValueFunction<>(0d);
-    this.nbBathroomsValueFunction = new ConstantValueFunction<>(0d);
-    this.terraceValueFunction = new ConstantValueFunction<>(0d);
-    this.floorAreaTerraceValueFunction = new ConstantValueFunction<>(0d);
-    this.wifiValueFunction = new ConstantValueFunction<>(0d);
-    this.pricePerNightValueFunction = new ConstantValueFunction<>(0d);
-    this.nbMinNightValueFunction = new ConstantValueFunction<>(0d);
-    this.teleValueFunction = new ConstantValueFunction<>(0d);
-
-    this.floorAreaSubjectiveValueWeight = 0.1;
-    this.nbBedroomsSubjectiveValueWeight = 0.1;
-    this.nbSleepingSubjectiveValueWeight = 0.1;
-    this.nbBathroomsSubjectiveValueWeight = 0.1;
-    this.terraceSubjectiveValueWeight = 0.1;
-    this.floorAreaTerraceSubjectiveValueWeight = 0.1;
-    this.wifiSubjectiveValueWeight = 0.1;
-    this.pricePerNightSubjectiveValueWeight = 0.1;
-    this.nbMinNightSubjectiveValueWeight = 0.1;
-    this.teleSubjectiveValueWeight = 0.1;
+    // value function
```

Comment:
J'enlèverai les commentaires //value function et //weight

Reply:
Fix in d34a55ad54fb6e348608d341481c740cbb7ff702